06-Customizing AzTS for your org/Extending AzTS/Readme.md

```diff
@@ -2,19 +2,24 @@
 <br>AzTS is not an official Microsoft product – rather an attempt to share Microsoft CSEO's best practices with the community.
 
 # Extending AzTS
-Azure Tenant Security Solution (AzTS) provides capability to add new controls(for existing services supported by AzTS) to customize the AzTS for your organization as per your need.
-Currently you can extend AzTS controls set by either adding a new control based on custom Azure Policy or based on Microsoft Defender for Cloud assessment using Control Metadata Editor Tool. 
 
-## Enabling Control Medatadata Editor tool
-This feature is not enabled by default. To enable this feature for your AzTS setup, please follow steps mentioned [here](Prerequisites.md#prerequisite-azts-configurations-to-enable-control-medatadata-editor-toolcmet).
+## Customizing AzTS for your org
 
-## Add new control for existing services
+### Control Metadata Editor tool
+CMET (Control Metadata Editor Tool) is a tool that can be used by privileged users (mostly by admins maintaining AzTS setup for organization)to customize AzTS for organization. It provides capabilities to update control metadata (Tags, Display Name, control settings(if applicable)), enable or disable control, etc.
```

Comment:
CMET (Control Metadata Editor Tool) is a tool designed for privileged users, typically administrators responsible for maintaining the AzTS setup for an organization. It allows customization of AzTS by providing features such as updating control metadata (tags, display name, control settings if applicable), enabling or disabling controls, and more.

Please note that this feature is disabled by default. To enable CMET for your AzTS setup, kindly follow the steps mentioned here:

Reply:
Updated